specifyweb/frontend/js_src/lib/components/Statistics/index.tsx

```diff
@@ -202,35 +216,65 @@ function ProtectedStatsPage(): JSX.Element | null {
   const sourceLayout = getSourceLayout(activePage.isShared);
 
   const allCategories = React.useMemo(
-    () => dynamicStatsSpec.map(({ responseKey }) => responseKey),
+    () => backEndStatsSpec.map(({ responseKey }) => responseKey),
```

Comment:
this file is way to be big and you keep adding to it. please think about splitting this component apart or factoring out logic into separate hooks.

Take a look at https://github.com/specify/specify7/blob/production/specifyweb/frontend/js_src/lib/components/Security/User.tsx for inspiration

Reply:
you added even more to this file 🧱